oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticDocumentMaker.java

```diff
@@ -113,11 +113,6 @@ protected void indexFulltextValue(ElasticDocument doc, String value) {
         doc.addFulltext(value);
     }
 
-    @Override
```

Comment:
Without this override, already analyzed properties will be duplicated in the `:fulltext` field.

The method should be changed in:

```java
/**
     * Avoid to persist in :fulltext field properties that are already analyzed since we can still query them
     * with a multi-match expression (see ElasticRequestHandler#fullTextQuery).
     */
    @Override
    protected boolean isFulltextValuePersisted(PropertyDefinition pd) {
        return !pd.analyzed;
    }
```